packages/cid-verifier/src/bindings.d.ts

```diff
@@ -0,0 +1,55 @@
+import Toucan from "toucan-js";
+import { Logging } from "@web3-storage/worker-utils/loki";
+
+export {};
+
+// CF Analytics Engine types not available yet
+export interface AnalyticsEngine {
+  writeDataPoint(event: AnalyticsEngineEvent): void;
+}
+
+export interface AnalyticsEngineEvent {
+  readonly doubles?: number[];
+  readonly blobs?: (ArrayBuffer | string | null)[];
+}
+
+export interface EnvInput {
+  ENV: string;
+  DEBUG: string;
+  GOOGLE_EVALUATE_SAFE_CONFIDENCE_LEVELS: Array<string>;
```

Comment:
Does this get in as a binding?

Reply:
looking after everything, it looks like we just not need this?

Reply:
Let;s sync on this than https://cloud.google.com/web-risk/docs/reference/rest/v1eap1/TopLevel/evaluateUri#confidencelevel